dbms/programs/performance-test/PerformanceTestSuite.cpp

```diff
@@ -321,10 +322,25 @@ std::unordered_map<std::string, std::vector<std::size_t>> getTestQueryIndexes(co
 int mainEntryClickHousePerformanceTest(int argc, char ** argv)
 try
 {
+    using namespace DB;
     using po::value;
-    using Strings = DB::Strings;
 
-    po::options_description desc("Allowed options");
+    unsigned line_length = po::options_description::m_default_line_length;
+    unsigned min_description_length = line_length / 2;
+    winsize terminal_size {};
+
+    if (isatty(STDIN_FILENO))
+    {
+        if (ioctl(STDIN_FILENO, TIOCGWINSZ, &terminal_size))
```

Comment:
Do you mind about making it a separate function instead of copy-paste?

Reply:
got this when tried to do so `/usr/bin/ld.gold: error: dbms/programs/performance-test/libclickhouse-performance-test-lib.a(PerformanceTestSuite.cpp.o): multiple definition of 'setOptionsDescription(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/bin/ld.gold: dbms/programs/client/libclickhouse-client-lib.a(Client.cpp.o): previous definition here`

Reply:
Is it OK if I make it `static`? Otherwise 
`/usr/bin/ld.gold: error: dbms/programs/performance-test/libclickhouse-performance-test-lib.a(PerformanceTestSuite.cpp.o): multiple definition of 'setOptionsDescription(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/bin/ld.gold: dbms/programs/client/libclickhouse-client-lib.a(Client.cpp.o): previous definition here
`

Reply:
No. You have to move function into .cpp to avoid ODR violation.